apps/gro/steps.js

```diff
@@ -152,7 +152,59 @@ module.exports = {
   '/confirm': {
     controller: require('./controllers/confirm'),
     template: 'confirm.html',
-    next: '/confirmation'
+    next: '/confirmation',
+    modelData: {
```

Comment:
Yeah maybe, I guess I was thinking of model as in Backbone/Marionette a model is tied to a template and the modelData is displayed. Meta would work well if it doesn't conflict elsewhere